File: sentimental-mario/mario.py
from cs50 import get_int
def main():
            while True:
                    n = get_int("Height")
                    if n > 0 and n < 9:
                         break
            for i in range(n):


                print((n - 1 - i) * " ", end="")
                print((i + 1) * "#")
main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)

{ int n;
do
{
  n = get_int("Height:");
}
while (n < 1 || n  > 8);
     int height;



{
    for (int i = 1; i < (n + 1); i++)

{
      for (int j = 1; j < (n + 1); j++)

        if (j <=(n - i))
        {
            printf(" ");
        }
        else
      {
          printf("#");
      }
      printf("\n");
  }

    }
    }



